drivers/soundwire/bus.c

```diff
@@ -1271,7 +1285,7 @@ static int sdw_initialize_slave(struct sdw_slave *slave)
 	val = prop->dp0_prop->imp_def_interrupts;
 	val |= SDW_DP0_INT_PORT_READY | SDW_DP0_INT_BRA_FAILURE;
 
-	ret = sdw_update(slave, SDW_DP0_INTMASK, val, val);
+	ret = sdw_update_no_pm(slave, SDW_DP0_INTMASK, val, val);
 	if (ret < 0)
 		dev_err(slave->bus->dev,
```

Comment:
@plbossart  should we change it to "dev_err(slave->dev,".  I see we change to use slave->dev now

Reply:
you're right @RanderWang we should also scan the dev_err/dbg log to see which device is used.

Reply:
done, can you recheck @RanderWang and @bardliao ?

Reply:
> done, can you recheck @RanderWang and @bardliao ?

@plbossart I checked bus.c. I think you didn't miss anything. But I didn't get why do we use bus->dev in ``sdw_modify_slave_status``?

Reply:
@bardliao there is only one case with dev_vdbg where we use bus->dev. I wanted to have one bus reference here that shows a bus-level evolution in terms of attachment, so that we can filter it out. It was really more to post-process the results than a functional change.

Reply:
> @bardliao there is only one case with dev_vdbg where we use bus->dev. I wanted to have one bus reference here that shows a bus-level evolution in terms of attachment, so that we can filter it out. It was really more to post-process the results than a functional change.

ok. got it. Thaks.